docs/Using-Fleet/REST-API.md

```diff
@@ -2537,6 +2537,7 @@ requested by a web browser.
 | Name                    | Type    | In    | Description                                                                                                                                                                                                                                                                                                                                 |
 | ----------------------- | ------- | ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | format                  | string  | query | **Required**, must be "csv" (only supported format for now).                                                                                                                                                                                                                                                                                |
+| columns                 | string  | query | **Required**, comma-delimited list of columns to include in the report (returns no result if no column is specified). |
```

Comment:
gotcha! thanks for the details, and please don't block the merge on this, was mostly a question for my own understanding :D

Reply:
Could we make it so if no columns are specified, all columns are returned (just like before, iiuc)? I haven't looked into this PR just yet, only this conversation, fwiw.

Reply:
Sure, no problem but that's a very good point that we should clarify. The endpoint is definitely broken if anyone used the old version given that it wouldn't return anything without the columns - another option would be to return all the columns if the field is not present, so we're backwards compatible. I wanted to be more on the cautions side by returning nothing if it is not present, but in this case it may be the wrong default.